proposal.h

```diff
@@ -0,0 +1,14 @@
+/* one */
+proto:
+int nrf_setdnsaddr(const char *dns, size_t len);
```

Comment:
How would you detect address family in this case?

Reply:
And if you specify in the API that you expect a string here, there's no need for a `len` field.